.github/release-a-package.md

```diff
@@ -0,0 +1,32 @@
+# How to release a new library version
+
+This documentation is intended for core Stardust contributors and describes `@stardust-ui/react` NPM package release process.
+
+## Prerequisities
+
+1. You need NPM account and npm configured with your token,
```

Comment:
nit: may suggest to remove commas (and use dots or semicolons) at the end of list entries